src/SFA.DAS.AssessorService.Web/Views/Application/Submitted.cshtml

```diff
@@ -24,7 +24,7 @@
 
                     <div class="govuk-panel govuk-panel--confirmation">
                         <h1 class="govuk-panel__title">
-                            Withdrawal notification submitted
+                            Withdrawal request submitted
```

Comment:
Content for notification > request also needs updating in the What happens next paragraph

Reply:
Fixed